index.html

```diff
@@ -8,6 +8,8 @@
     <meta charset="utf-8">
     <title>New Tab</title>
     <link rel="stylesheet" href="css/newTab.css">
+    <link rel="stylesheet" href="css/contentSearchUI.css">
+    <link rel="preconnect" id="preconnector" crossorigin="anonymous">
```

Comment:
happy to create this element on the fly, as speculative connect gets called, but not sure if there's ever a time when the default engine is clobbered. we could remove it from the DOM after the search is done though, since the speculative connect isn't needed after the search is performed.